integrations/cross_platform/yocto.rst

```diff
@@ -0,0 +1,213 @@
+
+Yocto SDKs are completely self-contained. The binaries are linked against their own copy of `libc`, which results in no dependencies on the
+target system.
+
+Then, you can start creating Conan packages setting up the environment of the Yocto SDK and running a :command:`conan create` command
+with a suitable profile with the specific architecture of the toolchain.
+
+For example, creating packages for `arch=armv8`:
+
+The profile will be:
+
+.. code-block:: text
+   :caption: *armv8*
+
+    [settings]
+    os_build=Linux
+    arch_build=x86_64
+    os=Linux
+    arch=armv8
+    compiler=gcc
+    compiler.version=8
+    compiler.libcxx=libstdc++11
+    build_type=Release
+
+Activate the SDK environment and execute the create command.
+
+.. code-block:: bash
+
+    $ source oe-environment-setup-aarch64-poky-linux
+    $ conan create . user/channel --profile armv8
+
+This will generate the packages using the Yocto toolchain. Now you can upload the binaries to an Artifactory server so share and reuse in
+your Yocto builds.
+
+.. note::
+
+    If you are using CMake to create the Conan packages, Yocto injects a toolchain that configures CMake to only search for libraries in the
+    rootpath of the SDK with `CMAKE_FIND_ROOT_PATH <https://cmake.org/cmake/help/v3.0/variable/CMAKE_FIND_ROOT_PATH.html#variable:CMAKE_FIND_ROOT_PATH>`_.
+    This is something that has to be patched in order to allow CMake to find libraries in the Conan cache:
+
+    .. code-block:: cmake
+       :caption: *sdk/sysroots/x86_64-pokysdk-linux/usr/share/cmake/OEToolchainConfig.cmake*
+
+        set( CMAKE_FIND_ROOT_PATH $ENV{OECORE_TARGET_SYSROOT} $ENV{OECORE_NATIVE_SYSROOT} )
+        set( CMAKE_FIND_ROOT_PATH_MODE_PROGRAM NEVER )
+        # COMMENT THIS: set( CMAKE_FIND_ROOT_PATH_MODE_LIBRARY ONLY )
+        # COMMENT THIS: set( CMAKE_FIND_ROOT_PATH_MODE_INCLUDE ONLY )
+        # COMMENT THIS: set( CMAKE_FIND_ROOT_PATH_MODE_PACKAGE ONLY )
+
+    You can read more about those variables here:
+
+      - `CMAKE_FIND_ROOT_PATH_MODE_LIBRARY <https://cmake.org/cmake/help/v3.0/variable/CMAKE_FIND_ROOT_PATH_MODE_LIBRARY.html>`_
+      - `CMAKE_FIND_ROOT_PATH_MODE_INCLUDE <https://cmake.org/cmake/help/v3.0/variable/CMAKE_FIND_ROOT_PATH_MODE_INCLUDE.html>`_
+      - `CMAKE_FIND_ROOT_PATH_MODE_PACKAGE <https://cmake.org/cmake/help/v3.0/variable/CMAKE_FIND_ROOT_PATH_MODE_PACKAGE.html>`_
+
+.. important::
+
+    We strongly recommend to use the Yocto's SDK toolchain to create packages as they will be built with the optimization flags suitable to
+    be deployed later to an image generated in a Yocto build.
+
+.. seealso::
+
+    You can `create your own Yocto SDKs <https://www.yoctoproject.org/docs/2.6/sdk-manual/sdk-manual.html#sdk-building-an-sdk-installer>`_
+    or download and use
+    `the prebuilt ones <http://downloads.yoctoproject.org/releases/yocto/yocto-2.6.2/toolchain/x86_64/>`_.
+
+Deploying Conan packages to a Yocto image
```

Comment:
Deploying an application.

Reply:
Explain later with an example of an application A that can depend on N conan packages (libraries).